client/src/pages/Profile.js

```diff
@@ -1,9 +1,12 @@
-import React, { useState, useEffect, useCallback } from "react";
+import React, { useState, useEffect, useCallback, useRef } from "react";
 import { makeStyles } from "@material-ui/core/styles";
 import { useHistory } from "react-router-dom";
 
 import { Button, TextField, Typography, Container } from "@material-ui/core";
 
+import EditIcon from "@material-ui/icons/Edit";
+import SendIcon from "@material-ui/icons/Send";
+
 import UploadImage from "../components/uploadImage";
 
 const axios = require("axios");
```

Comment:
can we use `import axios from "axios";`?

Reply:
Not a big deal but yes I swapped all the const axios to import axios.